src/xcode/ENA/ENA/Source/Services/PPAccessControl/Model/SRSError.swift

```diff
@@ -4,9 +4,21 @@
 
 import Foundation
 
+protocol SRSErrorAlertProviding: Error {
+	/// The SRS Error Alert type aims to share error message based on error codes.
+	/// The type is optional, because there are error cases of error enums, that hasn't a relationship to SRS
+	var srsErrorAlert: SRSErrorAlert? { get }
+}
+
+protocol ErrorCodeProviding: Error {
+	typealias ErrorCode = String
+	/// Error Code
+	var description: ErrorCode { get }
+}
```

Comment:
Could go into separate file(s) if they are not only conformed to by the SRSError

Reply:
👍